website/addons/googledrive/client.py

```diff
@@ -113,3 +113,12 @@ def folders(self, folder_id='root'):
             throws=HTTPError(401)
         )
         return res.json()['items']
+
+    def file_or_folder_metadata(self, fileId='root'):
```

Comment:
This method isn't actually used anywhere (aside from another method that isn't used anywhere). Removing